env-for-circle/develop.sh

```diff
@@ -14,3 +14,5 @@ export IRS_SUPERUSER_EMAIL="${IRS_SUPERUSER_EMAIL_DEV}"
 export USTC_ADMIN_PASS="${USTC_ADMIN_PASS}"
 export USTC_ADMIN_USER="${USTC_ADMIN_USER}"
 export ZONE_NAME="${ZONE_NAME_DEV}"
+export SLACK_WEBHOOK_URL=${SLACK_WEBHOOK_URL_DEV}
+export BOUNCE_ALERT_RECIPIENTS=${BOUNCE_ALERT_RECIPIENTS_DEV}
```

Comment:
I think you want to wrap these 2 new variables' values in quotes

Reply:
y'know, I'm not 10000% sure these are being used any longer since the move to Secrets Manager. I'm a little fuzzy on that.